daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -1522,3 +1527,74 @@ int 1
 		logResponse(t, &response)
 	}
 }
+
+func TestDryrunScratchSpace(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	a := require.New(t)
+
+	paySender, err := logic.AssembleString(`
```

Comment:
`paySender` is probably a cut n paste variable name from an inner pay test?

Reply:
ops, yes, fixed